### ISSUE TITLE
Custom titles for Important Hotkeys

### ISSUE THREAD

**Issue by YaLTeR** [YaLTeR/niri#847]
The idea is to let the users customize what's shown in Important Hotkeys:
- change titles, e.g. "Spawn `sh`" to "Increase volume"
- remove binds from the list
- add binds to the list

My idea is to have something like `Mod+Z show-in-hotkey-overlay="My Help Text" { spawn ...; }` which will:
- add the bind to the list if it's not there by default
- change the default title if it's there by default
- remove the bind from the list if you set `show-in-hotkey-overlay=false`

The property name is up for bikeshedding.

**Comment by thangisme** [comment]
Not related, but is there any config to disable this initial popup on login yet?

**Comment by YaLTeR** [comment]
Yes: https://github.com/YaLTeR/niri/wiki/Configuration:-Miscellaneous#hotkey-overlay

**Comment by YaLTeR** [comment]
Implemented in a605e7f6227448bbaa18932a251c9cbab9a07686
